code/script.js

```diff
@@ -1,14 +1,116 @@
 // Variables that point to selected DOM elements
 const chat = document.getElementById('chat')
+const form = document.getElementById('name-form');
+const inputSection = document.getElementById('input-wrapper');
 
 // If you need any global variables that you can use across different functions, declare them here:
+let myPizza = ""
+let mySize = ""
 
+// Containing two message, the user selection and confirmation from bot
+const selectSize = (size) => {
+mySize=size
+showMessage(size, 'user')
+showMessage(`Super Mario, you took the ${size} size for your ${myPizza}`, 'bot')
+
+showConfirmationMenu()
+}
+
+const showDropdownMenuInsteadOfPizzaButtons = () => {
+  inputSection.innerHTML = `
+  <div class="sizeOptions">
+  <select id="selectSize">
+    <option val="Small">Small</option>
+    <option val="Medium">Medium</option>
+    <option val="Large">Large</option>
+  </select>
+  <button id="sizeButton">Select</button>
+  </div>
+  `
+
+  //Default size is medium in the select above so we set the option to medium as well. This means that if we select the default then option should be medium.
+
+  /*let option="medium"
+  let sizeList= document.getElementById('selectSize')
+  sizeList.addEventListener('change', (event)=> {option = event.target.value});
+  }*/
+
+  option = document.getElementById('selectSize')
+  document.getElementById('sizeButton').addEventListener('click', () => selectSize(option.value))
+}
+
+const showConfirmationMenu = () => {
+  inputSection.innerHTML = `
+  <div class="confirmationOption">
+  <button id="acceptButton">Accept</button>
+  <button id="declineButton">Decline</button>
+  </div>
+  `
+
+  document.getElementById('acceptButton').addEventListener('click', () => confirmOrder())
+  document.getElementById('declineButton').addEventListener('click', () => declineOrder())
+}
+
+const confirmOrder = () => {
+  inputSection.innerHTML = ``
+  showMessage(`Great your pizza will be ready in 15 min`, 'bot')
+}
+
+const declineOrder = () => {
+  inputSection.innerHTML = ``
+  showMessage(`Hope you have a great`, 'bot')
+}
+
+const pizzaSize = (pizzaSizes) => {
+  myPizza=pizzaSizes
+  showMessage(pizzaSizes, 'user')
+  showMessage(`Fantastico, you choose ${pizzaSizes}! What size would you like?`, 'bot');
+
+ //This function invokes the function showDropdownMenuInsteadOfPizzaButton(the list you can choose size from)
+ showDropdownMenuInsteadOfPizzaButtons()
+}
+
```

Comment:
Naming of some Function could possibly be improved, for example "showDropdownMenuInsteadOfPizzaButtons". But it is obviously better with clearer longer names than too short or incomprehensible ones.